src/pages/[platform]/build-a-backend/datastore/customize-primary-keys/index.mdx

```diff
@@ -34,4 +34,4 @@ import flutter0 from '/src/fragments/lib/datastore/native_common/advanced-workfl
 
 import js0 from '/src/fragments/lib/datastore/native_common/advanced-workflows.mdx';
 
-<Fragments fragments={{ javascript: js0, 'react-native': js0 }} />
+<Fragments fragments={{ javascript: js0, angular: js0, nextjs: js0, react: js0, vue: js0 }} />
```

Comment:
Looks like we are losing `react-native` on this fragment, was that intentional? if so do we also need to remove it from the `platforms` list above?